compiler/pom.xml

```diff
@@ -70,6 +80,7 @@
               <exclude>com.google.auto.service:auto-service</exclude>
               <exclude>com.google.auto.value:auto-value</exclude>
               <exclude>com.google.dagger:dagger</exclude>
+              <exclude>com.google.guava</exclude>
```

Comment:
Yes, just remove that line.

(I have a local commit that un-shades Guava, it should hopefully merge cleanly with your change)